content/en/docs/languages/js/resources.md

```diff
@@ -184,7 +184,7 @@ $ docker run --rm -p 8080:8080 nodejs-otel-getting-started
 Listening for requests on http://localhost:8080
 DockerCGroupV1Detector found resource. Resource {
   attributes: {
-    'container.id': 'fffbeaf682f32ef86916f306ff9a7f88cc58048ab78f7de464da3c320ldb5c54'
+    'container.id': 'fffbeaf682f32ef86916f306ff9a7f88cc58048ab78f7de464da3c3201db5c54'
```

Comment:
Hmm, why is this changed?

Reply:
https://cloud-native.slack.com/archives/CJFCJHG4Q/p1737383287635539?thread_ts=1737382980.052069&cid=CJFCJHG4Q

Reply:
Fwiw, I really do try to provide reasonable descriptions in my commit messages, this one is:
* https://github.com/open-telemetry/opentelemetry.io/pull/6014/commits/daf181f66a2d25dc883add3db66241ed99ef79ac

Reply:
@jsoref Thanks for the context!